files/Aboot/boot0.j2

```diff
@@ -86,7 +86,7 @@ installer_image_path="$image_path/$installer_image"
 
 boot_config="$target_path/boot-config"
 
-cmdline_allowlist='crashkernel'
+cmdline_allowlist="crashkernel hwaddr_ma1"
```

Comment:
Why is this PR introducing a new option `hwaddr_ma1`?

Reply:
I can remove it, it's not related to this PR but to something else.